### PR TITLE
[TAB-17] feature/Created DAL Entries

### PR DESCRIPTION
**Pull request by Slayrak**

### CODE REVIEW

File: Services/Restaurant/src/TT.Restaurant/appsettings.json
Comment (Andertius) [inline Services/Restaurant/src/TT.Restaurant/appsettings.json]: I'd rather have the real connection string in the `appsettings.development.json` file. Here we can have a template for the string e.g. `server=server-name;database=db-name;uid=root;password=password`

Comment (Andertius) [reply Services/Restaurant/src/TT.Restaurant/appsettings.json]: we can have the dev appsettings be in git ignore or smth

Comment (Slayrak) [reply Services/Restaurant/src/TT.Restaurant/appsettings.json]: maybe it would be even better to create .env file with all the variables we need, and use that mechanism, because we will need one eventually

**Comment by Slayrak** [comment]
@Andertius added .env support, waiting for the approval

File: Services/Restaurant/src/TT.Restaurant/appsettings.json
Comment (Andertius) [reply Services/Restaurant/src/TT.Restaurant/appsettings.json]: @Slayrak please add some placeholder insread of your actual connection string here

Comment (Slayrak) [reply Services/Restaurant/src/TT.Restaurant/appsettings.json]: Done I guess

**Review by Andertius — DISMISSED** [review]

**Review by Andertius — APPROVED** [review]

**Pull request merged**
